File: 1929 concatenation-of-array/solution.go
func getConcatenation(nums []int) []int {
    res:=make([]int,2*len(nums))
    for i:=0;i<len(nums);i++ {
        res[i]=nums[i]
        res[i+len(nums)]=nums[i]
    }
    return res
}

File: 1929 concatenation-of-array/solution.rs
impl Solution {
    pub fn get_concatenation(nums: Vec<i32>) -> Vec<i32> {
        let n=nums.len();
        let mut res = vec![0; 2*n];
        for i in 0..nums.len() {
            res[i]=nums[i];
            res[i+n]=nums[i];
        }
        res
    }
}
